src/views/home/HomeViewProvider.ts

```diff
@@ -78,6 +79,30 @@ export class HomeViewProvider implements vscode.WebviewViewProvider {
     this._view = panel
   }
 
+  public async refreshAll() {
```

Comment:
this method is gonna cause the same login issue that was fixed yesterday. You'll have to add the same disposed check as this PR https://github.com/DevCycleHQ/vscode-extension/pull/168